react-dom: npm install --production error

For some specific reason I need to bundle react-dom with browserify. However I faced a problem with installing its' dependencies. Check my steps:

- download react-dom `.tar` file from npm registry (http://registry.npmjs.org/) and unpack it
- go to the react-dom folder and run `npm install --production` 

This results in an error:

```bash
react-dom dmitri$ npm install --production
npm ERR! Darwin 15.3.0
npm ERR! argv "node" "/usr/local/bin/npm" "install" "--production"
npm ERR! node v0.12.7
npm ERR! npm  v2.11.3

npm ERR! Cannot read property 'react' of undefined
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/dmitri/github/browserify-cdn-test/node_modules/react-dom/npm-debug.log
```

Interesting fact - if run `npm install` without `--production` flag it gets peer dependencies without errors.

Though if it's issue with peer deps why it returns SO weird error **`npm ERR! Cannot read property 'react' of undefined`**. 

Maybe it's related to react-dom packaging?

If you ask why I need this, check this url - https://wzrd.in/standalone/react-dom@latest and https://wzrd.in/ service.

--- a/packages/react-addons/package.json
+++ b/packages/react-addons/package.json
@@ -8,6 +8,7 @@
     "react-addon"
   ],
   "license": "BSD-3-Clause",
+  "dependencies": {},
   "peerDependencies": {
     "react": "^0.15.0-alpha.1"
   }

--- a/packages/react-dom/package.json
+++ b/packages/react-dom/package.json
@@ -12,6 +12,7 @@
     "url": "https://github.com/facebook/react/issues"
   },
   "homepage": "https://facebook.github.io/react/",
+  "dependencies": {},
   "peerDependencies": {
     "react": "^0.15.0-alpha.1"
   }

